package.json

```diff
@@ -56,7 +56,7 @@
     "react-i18next": "4.8.0",
     "react-native": "0.55.4",
     "react-native-background-timer": "2.0.0",
-    "react-native-calendar-events": "github:jitsi/react-native-calendar-events#dabfabc4bacc1424a8b93ebdda6f3820dee198cb",
```

Comment:
@saghul, I cannot seem to find the old commit (in order to view the git log between it and the new commit). Is it possible that you did a force push? If you did, could you please check your git reflog for the old commit. If you have it there, could you please restore it in this repo and then merge, not force push?

Reply:
Thank you very much! I appreciate that because it doesn't take us away from reproducible builds!